packages/image/rollup.config.js

```diff
@@ -0,0 +1,13 @@
+import buble from 'rollup-plugin-buble';
```

Comment:
is this right? Should it be `buble`?

Reply:
Yeah that's verbatim from the old project